primefaces/src/main/resources/META-INF/resources/primefaces/confirmpopup/confirmpopup.js

```diff
@@ -251,6 +254,15 @@ PrimeFaces.widget.ConfirmPopup = PrimeFaces.widget.DynamicOverlayWidget.extend({
     restoreButtons: function() {
         var $this = this;
         if ($this.cfg.global) {
+            if ($this.content.data('p-text')) {
```

Comment:
as alternative we could store all the attributes in the root element
but not sure whats better or not